clamdtop/clamdtop.c

```diff
@@ -801,8 +803,19 @@ static int make_connection(const char *soname, conn_t *conn)
     send_string(conn, "nIDSESSION\nnVERSION\n");
     free(conn->version);
     conn->version = NULL;
-    if (!read_version(conn))
-        return 0;
+    rv = read_version(conn);
```

Comment:
```suggestion

    rv = read_version(conn);
```

Let's add a new line here so that clang-format doesn't try to align the `=`.